cicd/pipelines/using-pipelines-as-code.adoc

```diff
@@ -0,0 +1,58 @@
+:_content-type: ASSEMBLY
+[id="using-pipelines-as-code"]
+= Using Pipelines as Code
+include::_attributes/common-attributes.adoc[]
+:context: using-pipelines-as-code
+
+toc::[]
+
+:FeatureName: Pipelines as Code
+include::snippets/technology-preview.adoc[]
+
+[role="_abstract"]
+Pipelines as Code allows cluster administrators and users with necessary privileges to define pipeline templates as part of source code Git repositories. When triggered by source code push or a pull request for the configured Git repository, it runs the pipeline and reports status.  
+
+[id="pac-key-features"]
+== Key features
+Pipelines as Code supports the following:
+
+* Pull request status. When iterating over a pull request, the status and control is executed on the platform hosting the Git repository.
+* GitHub Checks API to set the status of a pipeline run, including rechecks.
+* GitHub Pull Request and Commit events. 
+* Pull request actions in comments, such as `/retest`.
+* Git events filtering, and separate pipeline for each event.
+* Automatic task resolution in Pipelines (local tasks, Tekton Hub and remote URLs).
```

Comment:
```suggestion
* Automatic task resolution in Pipelines (local tasks, Tekton Hub, and remote URLs).
```
- Per IBM SG section on "Punctuation", use a comma to separate three or more items in a series.